servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -548,94 +499,94 @@ def test_get_job_aggregations(self):
 
             different_labels = {'different-label': 'different-value'}
 
-            labels_no_show = {'no-show': 'cannot-see-me'}
-
-            # Do not change the order of creating jobs since the labels are updated
-
-            # Create a running job that has same label but no test_token label
-            job_running_no_token = self.api_job_id(
-                self.start_job(
-                    'sleep 180',
-                    labels=labels,
-                    name=name,
-                    with_test_token=False))
-            # Create an aborted job
+            # Created a job to be aborted
             job_aborted = self.api_job_id(
-                self.start_job(
-                    'sleep 180',
-                    labels=labels,
-                    name=name,
-                    with_test_token=True))
-            # Create a failed job
+                self.start_job('sleep 180', labels=labels, name=name))
```

Comment:
since name doesn't matter anymore, perhaps just make the names descriptive instead, e.g. `name="agg-abort"`, this may be useful for manual debugging if these tests fail